### PR TITLE
Add source for documentation site

### PR DESCRIPTION
**Pull request by timdonnelly**

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "gatsby-starter-default",
+  "description": "Gatsby default starter",
+  "version": "1.0.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
```
Comment (zach-klippenstein) [inline docs/package.json]: Is this all correct?

File: docs/gatsby-config.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  pathPrefix: '/pages/~TDONNELLY/workflowdocs/master/browse',
```
Comment (davidapgar) [inline docs/gatsby-config.js]: This path prefix should probably be updated

File: docs/src/markdown/test.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
+---
```
Comment (davidapgar) [inline docs/src/markdown/test.md]: I think this was a test, should be removed.

File: docs/package.json (hunk 1 above)
Comment (timdonnelly) [reply docs/package.json]: It is now!

**Pull request merged**